src/lib/if-prop.ts

```diff
@@ -7,3 +7,6 @@ export const ifStyle = (
 ) => {
   return condition ? style : elseStyle
 }
+
+export const toPx = (size?: number | string): string | undefined =>
```

Comment:
It's ok to have such utility but I would say this one is to optimistic...
I'm thinking at:
- if a string use a regex (or something) to find unit ('px', 'rem', 'em')
- in case of rem or em, at least specify a `baseFontSize` parameter

some reference:
- https://github.com/mattdesl/parse-unit
- https://github.com/futurist/unit-to-px/
- https://github.com/KyleAMathews/convert-css-length
- https://github.com/mikolalysenko/to-px/

Reply:
Why do you want to know the units in case of string?